flask_saml_sso/sso.py

```diff
@@ -179,9 +205,16 @@ def sso(auth):
 
     Redirects user to IdP login page specified in metadata
     """
+    logger.info('SSO called')
+
     return_to = flask.request.args.get(
         'next', flask.request.host_url)
     login = auth.login(return_to=return_to)
+
+    logger.debug('SSO - RelayState: {}'.format(return_to))
```

Comment:
FWIW the logging module has some [built-in support](https://docs.python.org/3/library/logging.html#logging.debug) for formatting.

Reply:
From what I can tell, the formatting is set on a global level similar to `level`, and not on a per-logger basis. I've changed the messages to be slightly less verbose, though.